src/app/backend/replicasetpods.go

```diff
@@ -47,26 +54,31 @@ type ReplicaSetPodWithContainers struct {
 	// Time the Pod has started. Empty if not started.
 	StartTime *unversioned.Time `json:"startTime"`
 
+	// Total number of restarts.
+	TotalRestartCount int `json:"totalRestartCount"`
+
 	// List of Containers that belongs to particular Pod.
 	PodContainers []PodContainer `json:"podContainers"`
 }
 
 // Returns list of pods with containers for the given replica set in the given namespace.
-func GetReplicaSetPods(client *client.Client, namespace string, name string) (
+func GetReplicaSetPods(client *client.Client, namespace string, name string, limit string) (
```

Comment:
You should to Atoi in the caller function. The one that unpacks arguments. This one does not have any reason to accept strings. Only the code that glues API endpoints with this should do param marshalling/unmarshalling